docs/dashboard/dashboard.rst

```diff
@@ -0,0 +1,271 @@
+
+Finaly, clicking the version number will take us to the overview of that particular version.
+It gives the same options as the version overview does and it displays a list of all resources and their current state.
+
+.. figure:: /dashboard/images/img_8.png
+   :width: 100%
+   :align: center
+   :alt: Resource overiew
+
+Using the filters we can filter for resources by type, by agent used to deploy the resource, value and, maybe most importantly, by deploy state.
+This display is continuously updated, both during deploys and after, when the orchestrator goes through all resources to make sure they remain in the desired state.
+
+Taking a closer look at the a specific resource, there are 2 important buttons, the ``Dependency`` button and a ``magnifying glass``.
+The ``Dependency`` button is only available if a resource depends on toehr resources.
+When pressed it will add lines to the table dispalying the each dependency and it's current state:
+
+.. figure:: /dashboard/images/img_9.png
+   :width: 100%
+   :align: center
+   :alt: Resource dependencies
+
+   Resource dependencies
+
+The `magnifying glass` icon will take us to an in depth overview of the resource.
+This will show a complete breakdown of the resource's desired state at the top and.
+
+The desired state breakdown allows for easy inspection of the impact the resource will have.
+For example, the resource in the image below will deploy a file with path `/etc/my.cnf` and file permission `644`.
+We can even inspect the file's content.
+
+.. figure:: /dashboard/images/img_10.png
+   :width: 100%
+   :align: center
+   :alt: Resource view
+
+   Resource desired state view
+
+Below is the action log, shows a log of actions taken on the given resource.
+This varies from dry-runs to deploys.
+This log will typically start filling up with deploys due to the orchestrator enforcing the desired state.
+Again we can further inspect an action by pressing the drop down arrow.
+
+.. figure:: /dashboard/images/img_11.png
+   :width: 100%
+   :align: center
+   :alt: Resource action
+
+   Resource action view
+
+Each of these logs can then be further analyzed by pressing the ``magnifying glass``.
+
+The Resources overview
+----------------------
+
+The resources overview, not to be confused with the similar resource version overview, gives an overview of all known resources.
+This is not only for resources of the currently deployed model, but potentially resources from older models and the state they are in.
+
+.. figure:: /dashboard/images/img_12.png
+   :width: 100%
+   :align: center
+   :alt: Resources Overview
+
+   The Resources Overview
+
+While not as in depth as the resource version overview, it does link every resource to it's deployed version, so the resource can be inspected there.
+
+The Parameters View
+-------------------
+
+The parameter overview gives a list of parameters, pieces of information that are not part of the desired state model.
```

Comment:
I think we should phrase this differently. The parameters are part of the model, but their values may or may not be unknown at compilation time. For example, the IP address of a VM that has be be created.

Reply:
done